backend/routes/index.ts

```diff
@@ -78,6 +78,13 @@ router.post(
 );
 router.get('/comments/:locationId', CommentsController.getLocationComments);
 
+router.patch(
+  '/comments/:id',
+  CommentSchema,
```

Comment:
validation scheme for updating comment should be differrent from creating comment scheme(without likes dislikes etc.)

Reply:
I can make a separate scheme without problems but this makes no sense. Since in the future it will be necessary to update likes and dislikes and as a result, this scheme will differ in only one field.